### PR TITLE
Issue/1250 increase timeout for gdal http and updated url for loading gibs

### PR DESCRIPTION
**Pull request by micahnyc**
a value of 10 worked for me but I thought to make it 15 for EU users....

### CODE REVIEW

**Comment by micahnyc** [comment]
fixes #1250

**Comment by micahnyc** [comment]
want to check if this is affecting the loading of the tiles

File: modules/globebrowsing/globebrowsingmodule.cpp [hunk 1]
```diff
@@ -84,6 +84,7 @@
 #endif // _MSC_VER
 
 #include <cpl_string.h>
+#include <cpl_conv.h>
```
Comment (alexanderbock) [inline modules/globebrowsing/globebrowsingmodule.cpp:87]:
```suggestion
#include <cpl_conv.h>
#include <cpl_string.h>
```

File: modules/globebrowsing/globebrowsingmodule.cpp [hunk 2]
```diff
@@ -258,6 +259,8 @@ void GlobeBrowsingModule::internalInitialize(const ghoul::Dictionary& dict) {
             static_cast<size_t>(CpuCap.installedMainMemory() * 0.25 * 1024 * 1024)
         );
         addPropertySubOwner(GdalWrapper::ref());
+
+        CPLSetConfigOption("GDAL_HTTP_TIMEOUT", "15");
```
Comment (alexanderbock) [inline modules/globebrowsing/globebrowsingmodule.cpp:263]: Would be better to move this to the GDALWrapper class where we already have:

`    CPLSetConfigOption("GDAL_HTTP_TIMEOUT", "3"); // 3 seconds`

**Review by alexanderbock — CHANGES_REQUESTED** [review]

**Comment by alexanderbock** [comment]
Can you squash-merge the commits as the current commit messages are not standard (not starting with a capital letter, for example)

**Pull request merged**